### PR TITLE
New subworkflow vcf_impute_glimpse

### PR DESCRIPTION
**Pull request by LouisLeNezet**
## PR checklist

Here my first workflow for nf-core on glimpse imputation.

- [x] This comment contains a description of changes (with reason).
- [X] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [X] If necessary, include test data in your PR.
- [X] Remove all TODO statements.
- [X] Emit the `versions.yml` file.
- [X] Follow the naming conventions.
- [X] Follow the parameters requirements.
- [X] Follow the input/output options guidelines.
- [X] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [x] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [X] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Comment by LouisLeNezet** [comment]
Link to issue #2857

File: subworkflows/nf-core/vcf_impute_glimpse/meta.yml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  - chunk
+  - phase
+  - ligate
+
+modules:
+  - glimpse/chunk
+  - glimpse/phase
+  - glimpse/ligate
+
+input:
```
Comment (nvnieuwk) [inline subworkflows/nf-core/vcf_impute_glimpse/meta.yml:14]: Inputs in subworkflows should be seperated per input channel. You can describe in the descriptions of each channel what files should be in it (preferable with the structure), see here for an example: https://github.com/nf-core/modules/blob/master/subworkflows/nf-core/vcf_extract_relate_somalier/meta.yml

File: subworkflows/nf-core/vcf_impute_glimpse/meta.yml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        Sample names that are not present are assumed to have ploidy 2 (diploids).
+        GLIMPSE does NOT handle the use of sex (M/F) instead of ploidy.
+      pattern: "*.{txt,tsv}"
+
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test' ]
```
Comment (nvnieuwk) [inline subworkflows/nf-core/vcf_impute_glimpse/meta.yml:68]: Same for the output meta

**Review by nvnieuwk — CHANGES_REQUESTED** [review]
Good job, here are some improvement suggestions

File: modules/nf-core/glimpse/ligate/main.nf [hunk 3]
```diff
@@ -8,7 +8,7 @@ process GLIMPSE_LIGATE {
         'quay.io/biocontainers/glimpse-bio:1.1.1--hce55b13_1' }"
 
     input:
-    tuple val(meta), path(input)
+    tuple val(meta), val(input_list)
```
Comment (nvnieuwk) [inline modules/nf-core/glimpse/ligate/main.nf:11]:
```suggestion
    tuple val(meta), path(input_list)
```

This way the files get staged correctly (it's better when running with data stored on the cloud)

**Review by nvnieuwk — CHANGES_REQUESTED** [review]

**Review by nvnieuwk — APPROVED** [review]
LGTM

**Comment by LouisLeNezet** [comment]
Thanks a lot @nvnieuwk

**Pull request merged**